apps/hobbygroups/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from apps.hobbygroups import settings
+
+
+class Hobby(models.Model):
+    title = models.CharField(max_length=25)
+    description = models.CharField(max_length=250)
+    image = models.ImageField(upload_to=settings.IMAGES_PATH)
+    read_more_link = models.CharField(max_length=100, blank=True)
```

Comment:
This could probably be set as an URL field to gain the benefits of URL validation :-)